lib/ui/platform_dispatcher.dart

```diff
@@ -1012,6 +1022,44 @@ class PlatformDispatcher {
     );
   }
 
+  ErrorCallback? _onError;
+  Zone? _onErrorZone;
+
+  /// A callback that is invoked when an unhandled error occurs in Dart.
```

Comment:
So for `compute`, we forward errors back to the root isolate and rethrow them there. If a user does something else with spawning an isolate I'm not sure we need to cover that here.

Errors thrown in Dart as opposed to Java/Kotlin, Objective C/Swift, etc.

Reply:
Also, `addErrorListener` doesn't work for our use case here since so many times the Dart code getting invoked is coming from C++ `Dart_InvokeClosure` type calls, which don't end up bubbling exceptions up the same way in "dart code" (as opposed to the C++ embedding code).